docs/rules/id-length.md

```diff
@@ -31,6 +31,7 @@ var myObj = { a: 1 };
 class x { }
 class Foo { x() {} }
 function foo(...x) { }
+function foo({x}) { }
 var { x } = {};
 var { x: a} = {};
```

Comment:
It might be good to avoid confusion about which one is invalid, maybe we could rename `x` to have the correct length?